app/assets/stylesheets/omni_nav/omni_nav_v1/modules/omni-nav.scss

```diff
@@ -337,14 +337,14 @@ a.cu_nav_button {
         transition: background-color 0.5s ease-out;
         position: relative;
         line-height: 16px;
-        font-family: "proxima-nova-soft", "proxima-nova", "futura-pt", Arial, Verdana, "Sans Serif";
+        font-family: Arial, sans-serif;
```

Comment:
Still seeing changes in this file and `app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss`. Even though they're minor, I want them to stay the same as before, like it is on the development and master branches.

Reply:
@kaylaziegler I'm sorry about that Kayla – I reverted each individual line change I could find as opposed to copying the version from development. I guess I missed a few lines in doing it this way. Instead, I'm now copying the entire omni nav folder from the development branch and will replace the one in this branch.